config/dev.exs

```diff
@@ -28,15 +28,7 @@ config :flying_penguin, FlyingPenguinWeb.Endpoint,
   secret_key_base: System.get_env("FLYING_PENGUIN_KEY_BASE"),
   watchers: [
     # Start the esbuild watcher by calling Esbuild.install_and_run(:default, args)
-    esbuild: {Esbuild, :install_and_run, [:default, ~w(--sourcemap=inline --watch)]},
-    npx: [
-      "tailwindcss",
```

Comment:
At the risk of pointing out the obvious, The rest of this looks like config for the css framework. Removing this took out the tailwind integration and I guess we would just be using the css file that is put in here. Even though the styling does work a little better, it might be better to decide on a css framework and stick with it.

Reply:
I feel like discussing a css framework is outside of the scope of this PR. We can definitely discuss it later though

Reply:
I am commenting on it because you've proposed a change. If changing the framework is outside the scope of the PR then we shouldn't change it in this PR. It can't be both in the PR and outside the scope and unworthy of comment.

Reply:
Where you previously using tailwindcss? Because it looked like it wasn't doing anything.